Description: This patch let debhelper figures out proper compilation flags.
Author: Marcos Fouces <marcos@debian.org>
---
This patch header follows DEP-3: https://dep.debian.net/deps/dep3/
--- a/Makefile
+++ b/Makefile
@@ -4,8 +4,8 @@
 ###################################################################
 
 # Linux, FreeBSD, OpenBSD
-CFLAGS = 
-LDFLAGS = 
+#CFLAGS = 
+#LDFLAGS = 
 
 # Solaris / SPARC
 # CFLAGS = -DSOLARIS_SPARC
